main.tf

```diff
@@ -1,13 +1,77 @@
+# Optionally subscribe to Security Hub Standards 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards.html
+#-----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_standards_subscription" "this" {
+  for_each      = local.enabled_standards_arns
+  depends_on    = [aws_securityhub_account.this]
+  standards_arn = each.key
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Optionally configure Event Bridge Rules and SNS subscriptions 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-integration-types.html
+#-----------------------------------------------------------------------------------------------------------------------
+module "sns_topic" {
+  source = "git::https://github.com/cloudposse/terraform-aws-sns-topic.git?ref=tags/0.9.0"
+  count  = local.create_sns_topic ? 1 : 0
+
+  attributes      = ["securityhub-sns-topic"]
+  context         = module.this.context
+  subscribers     = {}
+  sqs_dlq_enabled = false
+}
+
+module "imported_findings_label" {
+  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.21.0"
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
-  }
+  attributes = ["securityhub-imported-findings"]
+  context    = module.this.context
 }
 
+resource "aws_cloudwatch_event_rule" "imported_findings" {
+  count       = local.enable_notifications == true ? 1 : 0
+  name        = module.imported_findings_label.id
+  description = "SecurityHubEvent - Imported Findings"
+  tags        = module.this.tags
+
+  event_pattern = jsonencode(
+    {
+      "source" : [
+        "aws.securityhub"
+      ],
+      "detail-type" : [
+        var.cloudwatch_event_rule_pattern_detail_type
+      ]
+    }
+  )
+}
+
+resource "aws_cloudwatch_event_target" "imported_findings" {
+  count = local.enable_notifications == true ? 1 : 0
+  rule  = aws_cloudwatch_event_rule.imported_findings[0].name
+  arn   = local.imported_findings_notification_arn
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Locals and Data References
+#-----------------------------------------------------------------------------------------------------------------------
 locals {
-  example = format("%v %v", var.example, join("", random_integer.example[*].result))
+  enable_notifications               = module.this.enabled && (var.create_sns_topic || var.imported_findings_notification_arn != null)
+  create_sns_topic                   = module.this.enabled && var.create_sns_topic
+  imported_findings_notification_arn = local.enable_notifications ? var.imported_findings_notification_arn != null ? var.imported_findings_notification_arn : module.sns_topic[0].sns_topic.arn : null
+  enabled_standards_arns = toset([
```

Comment:
`terraform fmt`

Reply:
My editor runs `fmt` every time I save and I just ran it manually. I didn't change anything with this block.